apps/client/src/app/pages/accounts/create-or-update-account-dialog/create-or-update-account-dialog.component.ts

```diff
@@ -79,4 +103,33 @@ export class CreateOrUpdateAccountDialog implements OnDestroy {
     this.unsubscribeSubject.next();
     this.unsubscribeSubject.complete();
   }
+
+  public autoCompleteCheck() {
+    const inputValue = this.accountForm.controls['platformId'].value;
+    const matchingEntry = this.platforms.find(
+      (platform) => platform.name === inputValue
```

Comment:
`({ name }) => { return name === inputValue; }`

Reply:
during testing, I encountered a bug with my previous implementation. I fixed it by checking the input value.

only for inputvalue of type string do the stuff (find, setting value)

inputValue can be of type platform, when a value from the list is selected by user interaction

Reply:
additional thoughts: this approach (matching by platform name) only works if we can guarantee that there are no multiple platforms with the same name.
or else the current implementation returns the first match from the given array